GitHubLabels.ps1

```diff
@@ -652,11 +639,111 @@ function Add-GitHubLabel
     .PARAMETER Issue
         Issue number to add the label to.
 
-    .PARAMETER LabelName
+    .PARAMETER Name
         Array of label names to add to the issue
 
-    .PARAMETER Replace
-        If supplied, will replace all of the labels on the issue with the provided labels.
+    .PARAMETER AccessToken
+        If provided, this will be used as the AccessToken for authentication with the
+        REST Api.  Otherwise, will attempt to use the configured value or will run unauthenticated.
+
+    .PARAMETER NoStatus
+        If this switch is specified, long-running commands will run on the main thread
+        with no commandline status update.  When not specified, those commands run in
+        the background, enabling the command prompt to provide status information.
+        If not supplied here, the DefaultNoStatus configuration property value will be used.
+
+    .EXAMPLE
+        Add-GitHubLabel -OwnerName PowerShell -RepositoryName PowerShellForGitHub -Issue 1 -Name $labels
+
+        Adds labels to an issue in the PowerShellForGitHub project.
+#>
+    [CmdletBinding(
+        SupportsShouldProcess,
+        DefaultParametersetName='Elements')]
+    [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSShouldProcess", "", Justification="Methods called within here make use of PSShouldProcess, and the switch is passed on to them inherently.")]
+    param(
+        [Parameter(ParameterSetName='Elements')]
```

Comment:
Are `$OwnerName` and `$RepositoryName` mandatory for their parameter set?

Reply:
Can you double-check the help documentation shows the correct parameter sets for each command you added/modified?